### DISCUSSION TITLE
How to handle multiple views

### DISCUSSION THREAD

**Discussion by acidtumbleweed** [neutralinojs/neutralinojs#390] [Q&A]
Hey all,

Just trying to understand how to handle multiple views. My main page is index.html but if I add a link to "index2.html" I get errors and it fails to navigate when running the app. Do I need to include those files somehow to make it work?

Thanks!

**Comment by shalithasuranga** [comment]
Could you please try v2. Now you will be able to load any HTML file from any directory.
